File: Codigo/Obligatorio/codigo/ArenaGestorFront/src/app/models/Concerts/ConcertResultConcertDto.ts
import { ConcertGetMusicalProtagonistDto } from "./ConcertGetMusicalProtagonistDto";
import { ConcertResultLocationDto } from "./ConcertResultLocationDto";

export class ConcertResultConcertDto {
    concertId: Number = 0;
    tourName: String = "";
    date: Date = new Date();
    ticketCount: Number = 0;
    price: Number = 0;
    protagonists: Array<ConcertGetMusicalProtagonistDto> = new Array();
    location: ConcertResultLocationDto = new ConcertResultLocationDto();
}

File: Codigo/Obligatorio/codigo/ArenaGestor/ArenaGestor.APIContracts/Concert/ConcertResultConcertDto.cs
using System;
using System.Collections.Generic;

namespace ArenaGestor.APIContracts.Concert
{
    public class ConcertResultConcertDto
    {
        public int ConcertId { get; set; }
        public string TourName { get; set; }
        public DateTime Date { get; set; }
        public int TicketCount { get; set; }
        public double Price { get; set; }
        public List<ConcertGetMusicalProtagonistDto> Protagonists { get; set; }
        public ConcertResultLocationDto Location { get; set; }
    }
}
